tests/test-server.js

```diff
@@ -0,0 +1,99 @@
+import Ember from 'ember';
```

Comment:
It would be a simple way to view the Pending, Resolved, and Rejected effects on the `records` and  `datasetState`. It could be considered overkill, but I'm imagining the whole test suite again as
- Requesting Records
  - Resolving Records
  - Rejecting Records